.github/workflows/push-deploy-website.yml

```
@@ -0,0 +1,66 @@
+      # This "name" has to be the repos' branch that contains
+      # the current active website. There must be an entry for
+      # the same branch in the PalisadoesFoundation's
+      # "Code and automation > Environments > github-pages"
+      # menu. The branch "name" must match the branch in the
+      # "on.push.branches" section at the top of this file
+      name: develop-postgres
+      url: https://docs-api.talawa.io
+    steps:
+      - uses: actions/checkout@v4
+      - uses: pnpm/action-setup@v4
+      - uses: actions/cache@v3
```

actionlint (1.7.7)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)